<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="rootIsDecorated">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Startup</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Acquisition</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>General</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="pageWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="Startup">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>When starting the application...</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QCheckBox" name="StartupAutoconnect">
                <property name="text">
                 <string>Autoconnect to the first device available</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Set to...</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QRadioButton" name="StartupSweepLastUsed">
                    <property name="text">
                     <string>Last used</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">StartupSweepGroup</string>
                    </attribute>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="StartupSweepDefault">
                    <property name="text">
                     <string>Default values</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">StartupSweepGroup</string>
                    </attribute>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>Vector Network Analyzer</string>
                </property>
                <layout class="QFormLayout" name="formLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Start:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="SIUnitEdit" name="StartupSweepStart"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Stop:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="SIUnitEdit" name="StartupSweepStop"/>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Simulus level:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QDoubleSpinBox" name="StartupSweepLevel">
                   <property name="suffix">
                    <string>dbm</string>
                   </property>
                   <property name="minimum">
                    <double>-42.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>0.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.250000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Points:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QSpinBox" name="StartupSweepPoints">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>4501</number>
                   </property>
                   <property name="value">
                    <number>501</number>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>IF bandwitdh:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="SIUnitEdit" name="StartupSweepBandwidth"/>
                 </item>
                 <item row="5" column="0">
                  <widget class="QLabel" name="label_18">
                   <property name="text">
                    <string>Averaging:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="1">
                  <widget class="QSpinBox" name="StartupSweepAveraging">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>99</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>Signal Generator</string>
                </property>
                <layout class="QFormLayout" name="formLayout_2">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Frequency:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="SIUnitEdit" name="StartupGeneratorFrequency"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Output level:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QDoubleSpinBox" name="StartupGeneratorLevel">
                   <property name="suffix">
                    <string>dbm</string>
                   </property>
                   <property name="minimum">
                    <double>-42.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>0.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.250000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_4">
                <property name="title">
                 <string>Spectrum Analyzer</string>
                </property>
                <layout class="QFormLayout" name="formLayout_3">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Start:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="SIUnitEdit" name="StartupSAStart"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>Stop:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="SIUnitEdit" name="StartupSAStop"/>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>RBW:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="SIUnitEdit" name="StartupSARBW"/>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_12">
                   <property name="text">
                    <string>Window:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QComboBox" name="StartupSAWindow">
                   <item>
                    <property name="text">
                     <string>None</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Kaiser</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Hann</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Flat Top</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="label_13">
                   <property name="text">
                    <string>Detector:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QComboBox" name="StartupSADetector">
                   <item>
                    <property name="text">
                     <string>+Peak</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>-Peak</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Sample</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Normal</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Average</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="6" column="0">
                  <widget class="QLabel" name="label_14">
                   <property name="text">
                    <string>Signal Identification:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="1">
                  <widget class="QCheckBox" name="StartupSASignalID">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0">
                  <widget class="QLabel" name="label_19">
                   <property name="text">
                    <string>Averaging:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="1">
                  <widget class="QSpinBox" name="StartupSAAveraging">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>99</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>17</width>
               <height>37</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>48</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Acquisition">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>Vector Network Analyzer</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QCheckBox" name="AcquisitionAlwaysExciteBoth">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If only S11/S21 or S22/S12 are enabled, faster sweeps are possible by only exciting one port. Checking this option forces the device to always excite both ports even when the measurements from one port will not be used.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Always perform full 2-port measurement</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="AcquisitionSuppressPeaks">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Due to limited fractional divider settings, the source and 1.LO PLLs are not able to reach every frequency exactly. At some specific frequencies this causes the final IF to shift. At these frequencies there will be a positive or negative peak in the trace measurement that is not actually there.&lt;br/&gt;&lt;br/&gt;Checking this option shifts the 2.LO for points where this could be an issue. This will remove the peaks but slows down the sweep slightly.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Suppress invalid peaks</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="AcquisitionUseHarmonic">
              <property name="text">
               <string>Use harmonic mixing</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>Spectrum Analyzer</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QFormLayout" name="formLayout_5">
              <item row="0" column="0">
               <widget class="QCheckBox" name="AcquisitionUseDFT">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Normally, the spectrum analyzer mode tunes the LO for each point and measures the final IF only at one frequency. When this option is enabled, a DFT of the final IF is calculated instead which covers multiple frequencies with one measurement.&lt;/p&gt;&lt;p&gt;This can speed up the measurement at low RBWs significantly.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Use DFT when RBW is below</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="SIUnitEdit" name="AcquisitionDFTlimitRBW"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="General">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QGroupBox" name="groupBox_5">
               <property name="title">
                <string>Graph colors</string>
               </property>
               <layout class="QFormLayout" name="formLayout_4">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>Background:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="ColorPickerButton" name="GeneralGraphBackground">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_16">
                  <property name="text">
                   <string>Axis:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="ColorPickerButton" name="GeneralGraphAxis">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_17">
                  <property name="text">
                   <string>Divisions:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="ColorPickerButton" name="GeneralGraphDivisions">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_8">
               <property name="title">
                <string>SCPI Control</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QCheckBox" name="GeneralSCPIEnabled">
                  <property name="text">
                   <string>Enable server</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QLabel" name="label_20">
                    <property name="text">
                     <string>Port:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="GeneralSCPIPort">
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>65535</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_9">
               <property name="title">
                <string>Hardware</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <widget class="QGroupBox" name="groupBox_10">
                  <property name="title">
                   <string>Temperature</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_10">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_8">
                     <item>
                      <widget class="QSpinBox" name="GeneralTemperatureWarn_Limit">
                       <property name="showGroupSeparator" stdset="0">
                        <bool>false</bool>
                       </property>
                       <property name="value">
                        <number>70</number>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_21">
                       <property name="text">
                        <string>Celsius</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QComboBox" name="GeneralTemperatureWarn_Action"/>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SIUnitEdit</class>
   <extends>QLineEdit</extends>
   <header>CustomWidgets/siunitedit.h</header>
  </customwidget>
  <customwidget>
   <class>ColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>CustomWidgets/colorpickerbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="StartupSweepGroup"/>
 </buttongroups>
</ui>
